src/components/chart/_macro.njk

```diff
@@ -95,16 +102,20 @@
                         "enabled": item.dataLabels if item.dataLabels else false
                     },
                     "connectNulls": item.connectNulls if item.connectNulls else false,
-                    "type": item.type if item.type and item.type == 'line' and params.chartType in supportedChartTypesWithLine else params.chartType
+                    "type": item.type if item.type and (item.type == 'line' and params.chartType in supportedChartTypesWithLine) or (item.type == 'scatter' and params.chartType in supportedChartTypesWithScatter) else params.chartType
                 }
             %}
             {# Use `set` tag to avoid printing the return value of extend #}
             {% set _ = extend(series, seriesItem) %}
         {% endfor %}
         {%
             set config = {
+                "chart": {
+                    "type": params.chartType,
+                    "inverted": params.isChartInverted if params.isChartInverted else false
```

Comment:
if `isChartInverted` is supplied and set to `false`, does this give the expected value? I found that for the `legend` parameter it unexpectedly would resolve to `true` in that case.

Reply:
It would also be worth adding tests around this similar to the ones I've added for the legend.

Reply:
If we assume the default for `isChartInverted` is `false` then this works as expected - the issue with the legend parameter was it defaulted to `true` which made the logic needed to test was more complex. I assume it is ok for it to default to `false` - can you add it to the `macro-options.md` file?